charts/sourcegraph-appliance/templates/service.yaml

```diff
@@ -1,18 +1,26 @@
-{{- if .Values.service.enabled -}}
 apiVersion: v1
 kind: Service
 metadata:
-  name: {{ include "sourcegraph-appliance.fullname" . }}
-  namespace: {{ .Values.namespace }}
+  annotations:
+    prometheus.io/port: "6060"
+    sourcegraph.prometheus/scrape: "true"
+    {{- if .Values.service.serviceAnnotations }}
+    {{- toYaml .Values.service.serviceAnnotations | nindent 4 }}
+    {{- end }}
   labels:
-    {{- include "sourcegraph-appliance.labels" . | nindent 4 }}
+    app: sourcegraph-appliance
+    deploy: sourcegraph
+    app.kubernetes.io/component: frontend
+    {{- if .Values.service.serviceLabels }}
+      {{- toYaml .Values.service.serviceLabels | nindent 4 }}
+    {{- end }}
+  name: sourcegraph-appliance
```

Comment:
I'm a bit confused - shouldn't this be _frontend's_ service, but pointed at an appliance web app pod label query? I thought the plan was to do that, with the appliance reconfiguring this service to point at frontend once the backend is standing up, but maybe I'm out of date 🙏

Reply:
No, you're absolutely right, that is the desired outcome. But in order to get to Maintenance UI we still need a service and an ingress. I think you're right that the name should be `sourcegraph-frontend` instead of `sourcegraph-appliance` since it is frontend's service

Reply:
Oh yeah for sure, sounds like we agree. To be clear I was suggesting that we rename these resources to be the same as those of frontend, then merge this.